imageprocessor/imageprocessor-core/src/main/java/pl/edu/uj/kimage/Image.java

```diff
@@ -0,0 +1,5 @@
+package pl.edu.uj.kimage;
+
+public class Image {
+    //TODO this is not finished
```

Comment:
ok, but you should add some informations about class responsibility, cause you created this class